### PR TITLE
Containerized Docker Application Lifecycle - 06.4

### PR DESCRIPTION
**Pull request by mvelosop**
## Summary

Creating a new PR to replace https://github.com/dotnet/docs/pull/10624

Copy & Paste as a new PR

@rpetrusha, just to try something different 😉

### CODE REVIEW

File: docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md [hunk 1]
```diff
@@ -19,77 +19,57 @@ As it pertains to Docker, you can chart life-cycle events and performance counte
...
 
-## Microsoft Operations Management Suite
+## Security, backup, and monitoring services
 
-[Operations Management Suite](https://microsoft.com/oms) is a simplified IT management solution that provides log analytics, automation, backup, and site recovery. Based on [queries](https://blogs.technet.microsoft.com/msoms/2016/01/21/easy-microsoft-operations-management-suite-search-queries/) in Operations Management Suite, you can raise [alerts](https://docs.microsoft.com/azure/operations-management-suite/operations-management-suite-monitoring-alerts) and set remediation via [Azure Automation](https://docs.microsoft.com/azure/automation/). It also seamlessly integrates with your existing management solutions to provide a single pane-of-glass view. Operations Management Suite helps you to manage and protect your on-premises and cloud infrastructure.
+There are many support chores with lots of details that you have to handle to ensure your applications and infrastructure are in top notch condition to support business needs, and the situation gets heavier in the microservices realm, so you need a way to have both high-level and detailed views when you need to take action.
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md]:
```suggestion
There are many support chores with lots of details that you have to handle to ensure your applications and infrastructure are in top notch condition to support business needs, and the situation becomes more complicated in the microservices realm, so you need a way to have both high-level and detailed views when you need to take action.
```

File: docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md [hunk 2]
```diff
@@ -19,77 +19,57 @@ As it pertains to Docker, you can chart life-cycle events and performance counte
...
 
-### [Operations Management Suite](https://microsoft.com/oms) Container Solution for Docker
+Azure has the tools to manage an provide a unified view of four critical aspects of both your cloud and on-premises resources:
```
Comment (rpetrusha) [inline docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/monitor-containerized-application-services.md]:
```suggestion
Azure has the tools to manage and provide a unified view of four critical aspects of both your cloud and on-premises resources:
```

**Review by rpetrusha — APPROVED** [review]
This looks good, @mvelosop. I've left two suggested changes.

**Comment by mvelosop** [comment]
@rpetrusha, I'm not sure about the statuses, added the `WIP` because one of the checks was expecting it, but WIP means it's not ready to merge 😣 I guess I'm missing something!

**Comment by rpetrusha** [comment]
@mvelosop, the status check is to prevent PRs that contain "WIP" in the title from being merged. (That's also true if a PR contains "WIP" in a commit message -- something that is a bad idea and requires that you later change the message in order to merge.) The WIP label has no effect on the status check.

Since you've addressed all of my suggestions, I'll merge.

**Comment by mvelosop** [comment]
Thanks for the explanation @rpetrusha!

**Pull request merged**